android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1376,10 +1394,25 @@ public static void switchToNextKeyboard(Context context) {
     String langName = kbInfo.get(KMManager.KMKey_LanguageName);
     String kFont = kbInfo.get(KMManager.KMKey_Font);
     String kOskFont = kbInfo.get(KMManager.KMKey_OskFont);
-    if (InAppKeyboard != null)
+    if (InAppKeyboard != null) {
       InAppKeyboard.setKeyboard(kbId, langId, kbName, langName, kFont, kOskFont);
-    if (SystemKeyboard != null)
+    }
+    if (SystemKeyboard != null) {
       SystemKeyboard.setKeyboard(kbId, langId, kbName, langName, kFont, kOskFont);
+    }
+  }
+
+  // TODO: Refactor InAppKeyboard / SystemKeyboard logic
+  public static KMKeyboard getKeyboard(KeyboardType keyboard) {
```

Comment:
This is actually returning a `KMKeyboard` (which is a `WebView`) right? It's just a little bit of an ambiguous name; perhaps we can call it `getKMKeyboard()` which then refers to the class name